ArcGISRuntimeSDKQt_CppSamples/Routing/OfflineRouting/OfflineRouting.h

```diff
@@ -0,0 +1,84 @@
+// [WriteFile Name=OfflineRouting, Category=Analysis]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifndef OFFLINEROUTING_H
+#define OFFLINEROUTING_H
+
+namespace Esri
+{
+namespace ArcGISRuntime
+{
+class Graphic;
+class GraphicsOverlay;
+class Map;
+class MapQuickView;
+class RouteParameters;
```

Comment:
you're including this so don't need the fwd declare

we also normally put includes above fwd declares